plugins/Actions/templates/_profileSummary.twig

```diff
@@ -11,7 +11,7 @@
                             url|trim|lower starts with 'data:' %}
                                 {{ url }}
                             {% else %}
-                                <a href="{{ url }}" rel="noreferrer noopener" target="_blank"
+                                <a href="{{ url|e('url') }}" rel="noreferrer noopener" target="_blank"
```

Comment:
haven't tested but think the URL escape should be usually used only on components, not on the whole URL? maybe double check those links still work when they include protocol etc

Reply:
from the docs

> url: escapes a string for the URI or parameter contexts. This should not be used to escape an entire URI; only a subcomponent being inserted.

Reply:
same for the other usages of this

Reply:
@diosmosis can you change the escaping?

Reply:
Updated the PR, added a `whole_url` filter that can be used to whitelist the url. Has to be used in conjunction w/ html_attr or `|raw` otherwise twig will automatically escape w/ `html`.